### PR TITLE
small updates

### PR DESCRIPTION
**Pull request by PeerHerholz**
This PR proposes a few changes to the docs:
- the `pyns` section of the `ecosystem page` now has a link to pyns
- the `FAQ page` now has the “click to show” feature for each question so that folks can see all questions at once and only check those they’re interested in
- the `bibliography page` now has respective screenshot as well
- the code parts of the `colab tutorial` were slightly adapted to make it more readable/clearer 
- the `usage link` of the `pyns` part was adapted to now link to the respective parts of the `pyns` docs

### CODE REVIEW

File: docs/cli/Neuroscout_CLI_Colab_Demo.ipynb [hunk 1]
```diff
@@ -10,7 +10,9 @@
...
+    "Specifically, this section will show you how you can run a sample pre-generated analysis with an already provided `id` or provide your own `analysis id`. To run your own `analysis`, copy your `id` into the field in the cell labelled (`_1`), set `Neuroscout` `Analysis` `ID_` and then `run` all of the `cells`. The provided here `id` will run `10 subjects` and `1 run` from the [Budapest dataset](https://neuroscout.org/dataset/27), and may take around `15 minutes`. Larger `analyses` will take longer due to the limited free resources. To minimize downloads, we've uploaded the `Budapest` dataset to Google Drive, but feel free to run analyses from other datasets by downloading data using the `get` command. \n",
+    "\n",
+    "In order to make this work here, we need to install a few things first. This includes [Neurodebain](https://neuro.debian.net/) (to easily get `DataLad`), [DataLad](https://www.datalad.org/) (to get the data, here the `Budapest` dataset), and of course [Neuroscout](https://neuroscout.org/)."
```
Comment (rbroc) [inline docs/cli/Neuroscout_CLI_Colab_Demo.ipynb]: small typo, should be "Neurodebian"

**Review by rbroc — APPROVED** [review]
LGTM :)

File: docs/cli/Neuroscout_CLI_Colab_Demo.ipynb (hunk 1 above)
Comment (adelavega) [inline docs/cli/Neuroscout_CLI_Colab_Demo.ipynb]:
```suggestion
    "In order to make this work here, we need to install a few things first. This includes [Neurodebian](https://neuro.debian.net/) (to easily get `DataLad`), [DataLad](https://www.datalad.org/) (to get the data, here the `Budapest` dataset), and of course [Neuroscout](https://neuroscout.org/)."
```

**Comment by adelavega** [comment]
Thanks!

**Review by adelavega — APPROVED** [review]

**Pull request merged**